Predictors/kPCA.py

```diff
@@ -0,0 +1,43 @@
+from sklearn.decomposition import KernelPCA
+from sklearn.feature_extraction.text import CountVectorizer
+
+from data.import_data import import_data
+from data.LemmaCountVectorizer import LemmaCountVectorizer
+from numpy import array
```

Comment:
with numpy it's standard to do:
`import numpy as np`
and use `np.array` instead of `array` further down

(pycharm has even been hardcoded to know np should by numpy)